### PR TITLE
Update Social Identity Providers documentation

### LINKED ISSUE #24601: Update Social Identity Providers documentation
### Description

Update configuration steps and pictures of the Social Identity Providers in the documentation.

### PR DESCRIPTION
**Pull request by tondrusk**
Resolves #24601 
- Update configuration steps of the Social Identity Providers
- Update pictures
- Sort Social Identity Providers alphabetically

### CODE REVIEW

**Comment by wojnarfilip** [comment]
This is optional and subjective. I kinda liked how the images previously showed also a navigation and active realm as it was giving user some additional context. On the other hand I can also see argument that the current images without navigation look cleaner.

Not sure if we have some general recommendations for this.

**Comment by tondrusk** [comment]
> This is optional and subjective. I kinda liked how the images previously showed also a navigation and active realm as it was giving user some additional context. On the other hand I can also see argument that the current images without navigation look cleaner.
> 
> Not sure if we have some general recommendations for this.

@wojnarfilip This is also currently inconsistent in the documentation. The OCP 3 and OCP 4 IdPs do not show navigation, unlike the rest of the social IdPs. I prefer the cleaner version, which shows just the important.

File: docs/documentation/server_admin/topics/identity-broker/social/microsoft.adoc [hunk 1]
```diff
@@ -11,10 +11,12 @@
 image:images/microsoft-add-identity-provider.png[Add Identity Provider]
 +
 . Copy the value of *Redirect URI* to your clipboard.
-. In a separate browser tab, https://account.live.com/developers/applications/create[create a Microsoft app].
-.. Click *Add URL* to add the redirect URL to the Microsoft app.
-.. Note the *Application ID* and *Application Secret*.
-. In {project_name}, paste the value of the *Application ID* into the *Client ID* field.
-. In {project_name}, paste the value of the *Application Secret* into the *Client Secret* field.
+. In a separate browser tab, hhttps://portal.azure.com/#view/Microsoft_AAD_RegisteredApps/CreateApplicationBlade/quickStartType~/null/isMSAApp~/false[create a Microsoft app].
```
Comment (wojnarfilip) [inline docs/documentation/server_admin/topics/identity-broker/social/microsoft.adoc:14]:
```suggestion
. In a separate browser tab, https://portal.azure.com/#view/Microsoft_AAD_RegisteredApps/CreateApplicationBlade/quickStartType~/null/isMSAApp~/false[create a Microsoft app].
```

Double h in https

**Review by wojnarfilip — CHANGES_REQUESTED** [review]

File: docs/documentation/server_admin/topics/identity-broker/social/microsoft.adoc (hunk 1 above)
Comment (tondrusk) [reply docs/documentation/server_admin/topics/identity-broker/social/microsoft.adoc:14]: Good catch! Thanks!

**Review by wojnarfilip — APPROVED** [review]
LGTM.

**Comment by ahus1** [comment]
Thank you for the review. Giving it some more time for @andymunro to review it. 

Is this something that should be backported to KC22?

**Comment by miquelsi** [comment]
> Is this something that should be backported to KC22?

Yes this should be backported to KC22, together with the testsuite changes.

**Review by ahus1 — APPROVED** [review]
Approving as of @wojnarfilip's review.

File: docs/documentation/server_admin/topics/identity-broker/social/facebook.adoc [hunk 2]
```diff
@@ -4,17 +4,41 @@
 
 .Procedure
 . Click *Identity Providers* in the menu.
-. From the `Add provider` list, select `Facebook`. {project_name} displays the configuration page for the Facebook identity provider.
+. From the `Add provider` list, select `Facebook`.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/identity-broker/social/facebook.adoc:7]:
```suggestion
. From the *Add provider* list, select *Facebook*.
```

File: docs/documentation/server_admin/topics/identity-broker/social/facebook.adoc [hunk 3]
```diff
@@ -4,17 +4,41 @@
...
-. Enter the https://developers.facebook.com/docs/facebook-login/guides/access-tokens[`Client ID` and `Client Secret`] values from your Facebook app into the `Client ID` and `Client Secret` fields in {project_name}.
+.Add a use case
+image:images/meta-add-use-case.png[Add a use case]
++
+.. Select `Other`.
++
+.Select an app type
+image:images/meta-select-app-type.png[Select an app type]
++
+.. Select `Consumer`.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/identity-broker/social/facebook.adoc:24]:
```suggestion
.. Select *Other*.
+
.Select an app type
image:images/meta-select-app-type.png[Select an app type]
+
.. Select *Consumer*.
```

File: docs/documentation/server_admin/topics/identity-broker/social/facebook.adoc [hunk 4]
```diff
@@ -4,17 +4,41 @@
...
++
+.. Fill in all required fields.
+.. Click *Create app*. Meta then brings you to the dashboard.
++
+.Add a product
+image:images/meta-add-product.png[Add Product]
++
+.. Click *Set Up* in the *Facebook Login* box.
+.. Select *Web*.
+.. Enter the *Redirect URI's* value into the `Site URL` field and click *Save*.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/identity-broker/social/facebook.adoc:37]:
```suggestion
.. Enter the *Redirect URI's* value into the *Site URL* field and click *Save*.
```

File: docs/documentation/server_admin/topics/identity-broker/social/facebook.adoc [hunk 5]
```diff
@@ -4,17 +4,41 @@
...
+.. In the navigation panel, select *App settings* - *Basic*.
+.. Click *Show* in the *App Secret* field.
+.. Note the `App ID` and the `App Secret`.
+. Enter the https://developers.facebook.com/docs/facebook-login/guides/access-tokens[`App ID` and `App Secret`] values from your Facebook app into the *Client ID* and *Client Secret* fields in {project_name}.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/identity-broker/social/facebook.adoc:41]:
```suggestion
.. Note the *App ID* and the *App Secret*.
. Enter the https://developers.facebook.com/docs/facebook-login/guides/access-tokens[*App ID* and *App Secret*] values from your Facebook app into the *Client ID* and *Client Secret* fields in {project_name}.
```

**Comment by ahus1** [comment]
@tondrusk - sorry for merging this too early. Please address  @andymunro's comments in a new PR once his review is complete.

File: docs/documentation/server_admin/topics/identity-broker/social/facebook.adoc [hunk 6]
```diff
@@ -4,17 +4,41 @@
...
 . Click *Add*
 
 . Enter the required scopes into the *Default Scopes* field. By default, {project_name} uses the `email` scope. See https://developers.facebook.com/docs/graph-api[Graph API] for more information about Facebook scopes.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/identity-broker/social/facebook.adoc:44]:
```suggestion
. Enter the required scopes into the *Default Scopes* field. By default, {project_name} uses the *email*  scope. See https://developers.facebook.com/docs/graph-api[Graph API] for more information about Facebook scopes. 
```

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks good, but the standard is to use *asterisks* for anything you see on the screen and `back ticks` for anything you type.
I made suggestions in the document, but realized it's better to just explain this guideline.

**Pull request merged**